packages/spark/base/inputs/_text-input.scss

```diff
@@ -27,139 +20,6 @@ $sprk-input-huge-box-shadow: 0 3px 10px 1px rgba(0, 0, 0, 0.08) !default;
   border-color: $sprk-text-input-focus-border-color;
 }
 
-// Applies to any huge input
-.sprk-b-InputContainer--huge .sprk-b-TextInput {
```

Comment:
Moved all this to the input_container.scss file as it no longer only applies to text inputs.